src/index.js

```diff
@@ -11,40 +11,48 @@ export default function routing(config: Config) {
     paths[key] = pathToRegexp.compile(config[key]);
   });
 
-  const getUrl = function(
+  const getUrl = (
     key: string,
     params?: { [string]: string },
     queryStrings?: { [string]: mixed },
-  ): string {
+  ): string => {
     const path = paths[key](params);
     if (queryStrings) {
       return `${path}?${qs.stringify(queryStrings)}`;
     }
     return path;
   };
 
-  const getPattern = function(key: string, relative?: boolean): string {
+  const getPattern = (
```

Comment:
Personally I'm not a massive fan of passing in booleans to functions, with a whole load of if statements inside. It generally increases complexity and can cause functions to steer away from their initial use. 

Maybe we could be more explicit by doing the following;

```
getReleativePattern(){
  const pattern = getPattern(...arguments);
  return pattern.substring()
}
```

Reply:
https://medium.com/@amlcurran/clean-code-the-curse-of-a-boolean-parameter-c237a830b7a3 is a good article that hits the points

Reply:
This makes sense to me. @cutandpastey thoughts?

Reply:
Before we make these changes can we just sit down and discuss this API first?